.github/workflows/ci.yml

```diff
@@ -0,0 +1,564 @@
+name: Alfresco Community Repo CI
+
+on:
+  pull_request:
+    branches:
+      - feature/**
+      - fix/**
+  push:
+    branches:
+      - feature/**
+      - fix/**
```

Comment:
`master` and `release/**`?

Reply:
I think we don't need to trigger build on push for `master` and `release/**` branches because the `master_release` workflow is responsible for triggering for these branches. The mentioned workflow reuses `ci` workflow to run required tests.